zcash_client_backend/src/data_api.rs

```diff
@@ -992,7 +992,7 @@ pub struct ScannedBundles<NoteCommitment, NF> {
 }
 
 impl<NoteCommitment, NF> ScannedBundles<NoteCommitment, NF> {
-    pub(crate) fn new(
```

Comment:
This might not be needed yet but i was wondering if it is a possibility.

Reply:
If these are needed, they should be made as a separate PR outside of the context of this one.